image/manifest.go

```diff
@@ -27,100 +27,72 @@ import (
 	"strings"
 	"time"
 
+	"github.com/opencontainers/image-spec/image/cas"
 	"github.com/opencontainers/image-spec/schema"
+	"github.com/opencontainers/image-spec/specs-go"
 	"github.com/pkg/errors"
+	"golang.org/x/net/context"
 )
 
 type manifest struct {
-	Config descriptor   `json:"config"`
-	Layers []descriptor `json:"layers"`
+	Config *specs.Descriptor  `json:"config"`
```

Comment:
On Mon, Aug 29, 2016 at 07:00:16PM -0700, Stephen Day wrote:
> >  type manifest struct {
> > -	Config descriptor   `json:"config"`
> > -	Layers []descriptor `json:"layers"`
> > +	Config *specs.Descriptor  `json:"config"`
> 
> No need for ptr here.

I expect this is another instance of [1].  If so, can we hash that
discussion out there?  If not, can you explain why you don't want a
pointer here and how that differs from why you don't want a pointer
argument for Engine.Put?

[1]: https://github.com/opencontainers/image-spec/pull/159#discussion_r76720093